signatures/xmss/xmss-sha2/param/xmss-sha2-10-256.yaml

```diff
@@ -0,0 +1,12 @@
+name: XMSS-SHA2_10_256
+security level:
+    classical: 256
+    quantum: 256
```

Comment:
How can these be the same? XMSS 256 should be 128-bit quantum secure, right?

Reply:
Of course, my bad. I even found an explicit source for it (RFC § 5)